docs/details/dashboards/dashboard-manage-dashboards.md

```diff
@@ -1,118 +1,117 @@
-# Manage dashboards
+# Manage PMM dashboards
 
+PMM offers various types of dashboards for monitoring and managing the performance of your databases:
 
-This section describes how to manage your PMM dashboards and the widgets on those dashboards, including:
+1. **Standard dashboards**: include database overviews, database-specific dashboards, query analytics, system-level metrics, replication, and high availability dashboards. These offer comprehensive insights into various aspects of your database environment.
+2. **Experimental dashboards**: newer, or less stable dashboards that usually introduce new metrics or visualizations or monitoring approaches.
+3. **Custom dashboards**: user-created dashboards tailored to specific metrics or needs, allowing you to combine data from different sources for customized monitoring and analysis.
 
-- Creating dashboard folders
+Standard and experimental dashboards are "provisioned dashboards", meaning they are:
 
-- Managing dashboard folders
+- created by PMM out of the box based on predefined configurations to ensure consistency across deployments.
+- protected from editing through the UI to prevent accidental modifications. Trying to modify these dashboards via the UI will result in a `Cannot save provisioned dashboard` error.
+- replaced during PMM upgrades, to ensure they are always up-to-date with the latest version. Any manual changes made by setting `allowUIUpdates: true` in `/usr/share/grafana/conf/provisioning/dashboards/default.yml` will be overwritten.
```

Comment:
```suggestion
```

Let's drop this completely, as we don't want users to modify Default dashboards at all. IF they'll find the solution - it's on them to maintain

Reply:
I think we need to keep it, just remove the part about settings.
```suggestion
- replaced during PMM upgrades, to ensure they are always up-to-date with the latest version. Any manual changes will be overwritten.
```